app/src/main/java/fr/free/nrw/commons/nearby/contract/NearbyParentFragmentContract.java

```diff
@@ -71,6 +71,9 @@
         LatLng getLastLocation();
 
         com.mapbox.mapboxsdk.geometry.LatLng getLastFocusLocation();
+
+        boolean isCurrentLocationMarkerVisible();
```

Comment:
this is a very suspect view function, the presenter should know about this and tell the view to do things instead of querying the view

Reply:
It is an informaion that we can get from the map, so I didn't get how to learn it without asking to view.

Reply:
Okay this 1500 line class is a bit much to ask for this change. Best practice would be to store all this information in the model or the presenter in this case because this screen doesn't have a model so I guess this is VP.

I'll resolve this conversation